extensions/blocks/map/deprecated/v1/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import { omit } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import save from './save';
+
+const attributes = {
+	align: {
+		type: 'string',
+	},
+	points: {
+		type: 'array',
+		default: [],
+	},
+	mapDetails: {
+		type: 'boolean',
+		default: true,
+	},
+	zoom: {
+		type: 'integer',
+		default: 13,
+	},
+	mapCenter: {
+		type: 'object',
+		default: {
+			longitude: -122.41941550000001,
+			latitude: 37.7749295,
+		},
+	},
+	mapStyle: {
+		type: 'string',
+		default: 'default',
+	},
+	markerColor: {
+		type: 'string',
+		default: 'red',
+	},
+	preview: {
+		type: 'boolean',
+		default: false,
+	},
+	scrollToZoom: {
+		type: 'boolean',
+		default: false,
+	},
+	mapHeight: {
+		type: 'integer',
+	},
+	showFullscreenButton: {
+		type: 'boolean',
+		default: true,
+	},
+};
+
+export default {
+	attributes,
+	migrate: oldAttributes => {
+		const className =
+			oldAttributes.className.replace( /is-style-[^ ]+/, '' ) +
```

Comment:
Is this because blocks are not supposed to have more than one `is-style-` class?

Reply:
Yup. It's not entirely self explanatory, so I've added a comment to explain what's going on.